packages/forma-36-website/src/components/Header.js

```diff
@@ -51,7 +51,7 @@ const Logo = () => (
     width="32px"
     height="32px"
     viewBox="0 0 90 90"
-    enable-background="new 0 0 90 90"
+    enableBackground="new 0 0 90 90"
```

Comment:
Doh! Good spot

Reply:
TBH, I found it following a warning in the console hahaha there some other ones, but we can fix in other PR I think 😄